packages/web-app-files/src/components/Shares/SharedWithMeSection.vue

```diff
@@ -18,8 +18,7 @@
       v-else
       v-model:selectedIds="selectedResourcesIds"
       :data-test-share-status="shareStatus"
-      class="files-table"
```

Comment:
I bet this is being used as a selector in the nightwatch acceptance tests 🙈

Reply:
Yes, that's why I added it in the component... and you realize the tests are already green? 😇

Reply:
duuuuude! nvm then 😇